eras/conway/impl/src/Cardano/Ledger/Conway/Translation.hs

```diff
@@ -93,7 +93,7 @@ instance Crypto c => TranslateEra (ConwayEra c) ShelleyGenesis where
           API.sgUpdateQuorum = API.sgUpdateQuorum genesis,
           API.sgMaxLovelaceSupply = API.sgMaxLovelaceSupply genesis,
           API.sgProtocolParams = translateEra' ctxt (API.sgProtocolParams genesis),
-          API.sgGenDelegs = API.sgGenDelegs genesis,
+          API.sgGenDelegs = API.sgGenDelegs genesis,    -- TODO Is this correct?
```

Comment:
@JaredCorduan Could you please comment on this. I haven't fully groked what ConwayEra introduces yet.

Reply:
The conway era will probably only provide the master key rotation. For mainnet, and other networks that start with the Byron era, I do not know of any need to modify the master keys inside the Shelley genesis, but I am not an expert on everything that the consensus layer uses the various ledger state for. It's probably true that shelley-only networks do need to modify `sgGenDelegs`. I made #2988 for exactly these questions.

Reply:
Ok, I removed the TODO. If we need to do anything about it, we can do it in a follow up PR

Reply:
perfect, thanks